std/functional.d

```diff
@@ -305,7 +304,6 @@ private uint _ctfeSkipName(ref string op, string name)
 private uint _ctfeMatchUnary(string fun, string name)
 {
     if (!__ctfe) assert(false);
-    import std.stdio;
```

Comment:
This unused import was causing a shared ctor cycle after changing the imports in `std.range`.